client/my-sites/checkout/checkout-thank-you/index.tsx

```diff
@@ -654,7 +653,6 @@ export class CheckoutThankYou extends Component<
 				{ this.isDataLoaded() && this.isRedesignV2() && <ConfettiAnimation delay={ 1000 } /> }
 				{ this.isRedesignV2() && this.props.selectedSite?.ID && (
 					<>
-						<QuerySitePurchases siteId={ this.props.selectedSite.ID } />
```

Comment:
@candy02058912 Are you able to confirm the ProductPlan component was the only thing relying on this?

Reply:
Yeah, I added that in just for the expiry date purpose.

Reply:
Ah right, that was my first attempt but I realised it was loading all purchases and not necessarily the purchase for the current receipt so I decided to hide instead. 

I guess there is no way these could be different in this situation for plans produces so maybe I over thought it? :thinking: